File: xposed-stub/src/main/java/de/robv/android/xposed/IXposedHookZygoteInit.java
package de.robv.android.xposed;

public interface IXposedHookZygoteInit extends IXposedMod {
    void initZygote(StartupParam startupParam) throws Throwable;

    final class StartupParam {
        public String modulePath;
        public boolean startsSystemServer;
    }
}

File: app/src/main/java/io/github/magisk317/relay/xp/compat/IXposedHookZygoteInit.kt
package io.github.magisk317.relay.xp.compat

interface IXposedHookZygoteInit {
    @Throws(Throwable::class)
    fun initZygote(startupParam: StartupParam)

    class StartupParam {
        var modulePath: String? = null
        var startsSystemServer: Boolean = false
    }
}
